docs/e2e.md

```diff
@@ -0,0 +1,77 @@
+# E2E testing Guide
+
+> SkyWalking Infra E2E is the next generation End-to-End Testing framework that aims to help developers to set up, debug, and verify E2E tests with ease. It’s built based on the lessons learnt from tens of hundreds of test cases in the SkyWalking main repo.
+
+Currently, we implement our E2E testing by using [SkyWalking Infra E2E](https://github.com/apache/skywalking-infra-e2e). If you want to learn more, please refer to it's [documentation website](https://skywalking.apache.org/docs/skywalking-infra-e2e/latest/readme/).
+
+## The Structure of the E2E Cases
+
+```bash
+.github
+├── build-for-e2e
+│   └── action.yaml
+├── e2e
+│   └── action.yaml
+├── release-drafter.yml
+└── workflows
+    ├── backup.yaml
+    ├── build.yaml
+    ├── e2e.install.yaml # E2E testing workflow
+    ├── e2e.others.yaml  # E2E testing workflow
+    ├── lint.yaml
+    ├── nightly-build.yaml
+    └── release-drafter.yml
```

Comment:
How about just skipping those not-related files? Such as `nightly-build.yaml`, `release-drafter.yml`.

We hope people can understand the structure of e2e instead of the whole project.

Reply:
Done.